src/mlpack/core/data/detect_file_type.cpp

```diff
@@ -27,18 +27,18 @@ namespace data {
  *
  * @param type Type to get the logical name of.
  */
-std::string GetStringType(const arma::file_type& type)
+std::string GetStringType(const file_type& type)
 {
-  switch (type)
+   switch (type)
```

Comment:
```suggestion
  switch (type)
```
Seems like the extra space shouldn't be needed?

Reply:
@heisenbuug this one has not been resolved, just a reminder